crates/rolldown/src/utils/parse_to_ast.rs

```diff
@@ -41,6 +41,7 @@ pub fn parse_to_ast(
     ModuleType::Binary => {
       (binary_to_esm(&source, options.platform, ROLLDOWN_RUNTIME_RESOURCE_ID).into(), ParseType::Js)
     }
+    ModuleType::Empty => ("export default {}".to_string().into(), ParseType::Js),
```

Comment:
`export {}` not `export default {}`

Reply:
when I set ` export { }`,it will get output  
<img width="836" alt="image" src="https://github.com/rolldown/rolldown/assets/17617116/5af37f41-336b-40df-85ab-0a1789f98ec7">
so I changed `export default { }`